pkg/controller/pod_volume_restore_controller.go

```diff
@@ -258,9 +261,12 @@ func (c *podVolumeRestoreController) processRestore(req *velerov1api.PodVolumeRe
 	var err error
 
 	// update status to InProgress
-	req, err = c.patchPodVolumeRestore(req, updatePodVolumeRestorePhaseFunc(velerov1api.PodVolumeRestorePhaseInProgress))
+	req, err = c.patchPodVolumeRestore(req, func(r *velerov1api.PodVolumeRestore) {
+		r.Status.Phase = velerov1api.PodVolumeRestorePhaseInProgress
+		r.Status.StartTimestamp.Time = c.clock.Now()
+	})
 	if err != nil {
-		log.WithError(err).Error("Error setting phase to InProgress")
+		log.WithError(err).Error("Error setting startTimestamp and phase to InProgress")
```

Comment:
See similar error in the backup file - I'd mention podvolumerestore in the error.